### ISSUE TITLE
Dont set HIP_VISIBLE_DEVICES and ROCR_VISIBLE_DEVICES

### ISSUE THREAD

**Issue by IMbackK** [mcmonkeyprojects/SwarmUI#565]
### Expected Behavior

ROCR_VISIBLE_DEVICES ids and HIP device are redundant and dont refer to the same ids, ROCR uses the order supplied by the kernel drm interface while HIP uses the init order from rocr, which need not be the same as rocr will initialize gpus of the same architecture together

thus 
```
export ROCR_VISIBLE_DEVICES=1
export HIP_VISIBLE_DEVICES=1
```

can end up with _no_ device available since you have masked away all gpus

i would recommend using only  ROCR_VISIBLE_DEVICES as this works in all cases.

### Actual Behavior

can end up with _no_ device available since you have masked away all gpus

i would recommend using only  ROCR_VISIBLE_DEVICES as this works in all cases.

### Steps to Reproduce

Have a machine with n gfx908 gpus and z gfx1030 gpus, be unlucky with the initialization order and try 

```
$ export ROCR_VISIBLE_DEVICES=1
$ export HIP_VISIBLE_DEVICES=1
$ python
>>> import torch
>>> torch.cuda.is_available()
False
```

### Debug Logs

N/A
